pkgs/applications/audio/goattracker/default.nix

```diff
@@ -35,15 +35,15 @@ stdenv.mkDerivation (finalAttrs: {
     if isStereo then
       "2.77" # stereo
     else
-      "2.76"; # normal
+      "2.77"; # normal
 
   src = fetchurl {
     url = "mirror://sourceforge/goattracker2/GoatTracker_${finalAttrs.version}${lib.optionalString isStereo "_Stereo"}.zip";
     sha256 =
       if isStereo then
         "1hiig2d152sv9kazwz33i56x1c54h5sh21ipkqnp6qlnwj8x1ksy" # stereo
```

Comment:
This hash should also be updated

Reply:
Nevermind! While usually they get updated in lockstep, this time we only have a regular goattracker update.